doc/release-notes.md

```diff
@@ -11,6 +11,7 @@
       - `sample_id` and `aliquot_id` for some TARGET samples were miscoded before. This release fixed the issue per [ticket 145](https://github.com/PediatricOpenTargets/ticket-tracker/issues/145)
       - `broad_histology` for TARGET samples were updated as following: `Acute Lymphoblastic Leukemia` and `Acute Myeloid Leukemia` were merged to `Hematologic malignancies`; `Clear cell sarcoma of the kidney`, `Rhabdoid tumor`, and `Wilms tumor` were combined as `Renal tumors`; `Osteosarcoma` is changed to `Mesenchymal, non-meningothelial tumors` and `Neuroblastoma` is converted to `Embryonal tumors`. See [ticket 136](https://github.com/PediatricOpenTargets/ticket-tracker/issues/136)
       - For `gtex_group == "Cells"`, the `composition` column is changed from `Solid Tissue` to `Derived Cell Line` per discussion in d3b center [ticket 43](https://github.com/d3b-center/D3b-codes/pull/43)
+      - For `short_histology`, `NBL` is converted to `Neuroblastoma` to be consistent with other samples 
```

Comment:
```suggestion
      - For `short_histology`,  neuroblastoma samples previously annotated as `NBL` or `Embryonal tumor` are converted to `Neuroblastoma` to be consistent with other samples 
```

Reply:
For some reason, I cannot change line 8 in release notes:

- deiltaed changes: 
should be - `detailed changes:`

Reply:
Fixed!